Libplanet/Net/Swarm.cs

```diff
@@ -1161,6 +1174,11 @@ CancellationToken cancellationToken
 
             if (IsUnknownPeer(sender))
             {
+                if (IsDifferentProtocolVersion(sender))
+                {
+                    return;
```

Comment:
Is it ok to return without calling `DifferentVersionPeerEncountered`?

Reply:
I thought only when the swarm try to add another peer, but I think it would be good to call the handler in this case too.